01week/rockPaperScissors.js

```diff
@@ -8,18 +8,29 @@ const rl = readline.createInterface({
 });
 
 
-function rockPaperScissors(hand1, hand2) {
-
+function rockPaperScissors(inputHand1, inputHand2) {
+const hand1 = inputHand1.trim().toLowerCase()
+const hand2 = inputHand2.trim().toLowerCase()
+console.log(hand1, hand2)
   // Write code here
+  if ((hand1 === 'rock' && hand2 === 'paper') || (hand1 === 'paper' && hand2 === 'scissors') || (hand1 === 'scissors' && hand2 === 'rock')) {
```

Comment:
You're checking the same hand over and over, what's a better and more efficient way to check for a win?